src/PKSim.Presentation/Presenters/AboutPresenter.cs

```diff
@@ -31,7 +31,7 @@ public AboutPresenter(IAboutView view, IPKSimConfiguration configuration, IVersi
       public override void Initialize()
       {
          _view.Product = _configuration.ProductDisplayName;
-         _view.VersionInformation = $"Version {_configuration.FullVersionDisplay}";
+         _view.VersionInformation = $"Version {_configuration.FullVersion}";
```

Comment:
This will show the following
![image](https://user-images.githubusercontent.com/1041237/57816847-e08af880-774a-11e9-97f0-c0c55ff24718.png)

Reply:
@msevestre what about the build number?

Reply:
Build number is the last digit. 
8.5.4 => Version 8, Update 5 build 4